recipes/libjpeg/all/test_package/conanfile.py

```diff
@@ -8,13 +8,14 @@ class TestPackageConan(ConanFile):
 
     def build(self):
         cmake = CMake(self)
+        cmake.verbose = True
```

Comment:
### 
```suggestion
```

I think normally verbose is not set. Do we need it here?

Reply:
No, it isn't needed.
I added it to debug the commands.